controllers/osbuildenvconfig_controller.go

```diff
@@ -278,6 +344,106 @@ func (r *OSBuildEnvConfigReconciler) Update(ctx context.Context, reqLogger logr.
 		return ctrl.Result{Requeue: true}, nil
 	}
 
+	created, err = r.ensureWorkerConfigAnsibleConfigExists(ctx, instance)
+	if err != nil {
```

Comment:
those 6 lines can be exposed to a function - for example if you'll decide to requeueAfter a period time,
something like that: 
func (r *OSBuildEnvConfigReconciler)  <WHATEVER-NAME>(created boolean, err error, msg string) (ctrl.Result, error){
       if err != nil {
		return ctrl.Result{Requeue: true}, err
	} else if created {
		reqLogger.Info(msg)
		return ctrl.Result{Requeue: true}, nil
	}
}